auth/authenticator.go

```diff
@@ -59,9 +59,10 @@ func (l *LsatAuthenticator) Accept(header *http.Header, serviceName string) bool
 //
 // NOTE: This is part of the Authenticator interface.
 func (l *LsatAuthenticator) FreshChallengeHeader(r *http.Request,
-	serviceName string) (http.Header, error) {
+	serviceName string, servicePrice int64) (http.Header, error) {
 
-	service := lsat.Service{Name: serviceName, Tier: lsat.BaseTier}
+	service := lsat.Service{Name: serviceName, Tier: lsat.BaseTier,
```

Comment:
nit: if the struct initialization doesn't fit on one line, the preferred style is this one:
```
service := lsat.Service{
	Name: serviceName,
	Tier: lsat.BaseTier,
	Price: servicePrice,
}
```